front/app/modules/free/native_surveys/admin/components/SurveyFields/index.tsx

```diff
@@ -27,48 +27,52 @@ interface SurveyFieldsProps {
   surveyCustomFields: IFlatCustomField[];
   handleDragRow: (fromIndex: number, toIndex: number) => void;
   handleDropRow: (fieldId: string, toIndex: number) => void;
+  selectedFieldId?: string;
 }
 
 const SurveyFields = ({
   onEditField,
   surveyCustomFields,
   handleDragRow,
   handleDropRow,
+  selectedFieldId,
 }: SurveyFieldsProps) => {
   return (
     <DndProvider backend={HTML5Backend}>
       <Box p="32px" height="100%" overflowY="auto">
         <List key={surveyCustomFields.length}>
           {surveyCustomFields.map((field, index) => {
+            const border =
+              selectedFieldId === field.id ? `1px solid #74B4FF` : 'none';
```

Comment:
Just want to double check we don't have a named colour we could use here instead?

Reply:
I checked https://citizenlabdotco.github.io/cl2-component-library/?path=/story/design-colors--default-story and there isn't one. What do you normally do in these cases?

Reply:
Hmm.. maybe we could use one of the ClBlue colours for now, and make a note to discuss with Ben when he is back?

Reply:
Good idea, let me update this